### PR TITLE
Add DM to sponsors

### PR DESCRIPTION
**Pull request by GregSutcliffe**
And another one :)

@timogoebel is DM ok for company name, or do you prefer the longer title?

### CODE REVIEW

File: sponsors.md [hunk 1]
```diff
@@ -51,14 +51,19 @@ Thank you to NETWAYS, who provide servers for our continuous integration system.
...
 ### ATIX
 
-<div class="lead text-left"><a href="https://www.atix.de"></a><img src="/static/images/sponsors/atix.png" height="125px"/></div>
+<div class="lead text-left"><a href="https://www.atix.de"><img src="/static/images/sponsors/atix.png" height="125px"/></a></div>
 Thank you to ATIX, for their many ongoing contributions to the codebase.
 
+### DM
+
+<div class="lead text-left"><a href="https://www.dm.de"><img src="/static/images/sponsors/dm.jpg" height="125px"/></a></div>
+Thank you to DM, for their many ongoing contributions to the codebase.
```
Comment (timogoebel) [inline sponsors.md]: I'd prefer `dm-drogeriemarkt` here.

Comment (GregSutcliffe) [reply sponsors.md]: Is that as shown, without caps?

Comment (timogoebel) [reply sponsors.md]: On second thought, why not just use `dm` (lowercase on purpose). :-)

Comment (GregSutcliffe) [reply sponsors.md]: @timogoebel cool, updated

**Review by tbrisker — APPROVED** [review]
Thanks @GregSutcliffe  and @timogoebel !

**Comment by lzap** [comment]
It only took how many years? ;-) Thanks!

**Pull request merged**